packages/node/src/dev-server.ts

```diff
@@ -124,14 +119,14 @@ export async function onDevRequest(
   }
 
   try {
-    const result = await handleEvent(req);
-    res.statusCode = result.statusCode;
-    for (const [key, value] of Object.entries(result.headers)) {
-      if (typeof value !== 'undefined') {
+    const { headers, body, status } = await handleEvent(req);
+    res.statusCode = status;
+    for (const [key, value] of headers as Headers) {
+      if (value !== undefined) {
         res.setHeader(key, value);
       }
     }
-    res.end(Buffer.from(result.body, result.encoding));
+    toToReadable(body).pipe(res);
```

Comment:
Why are we not using: https://nodejs.org/api/stream.html#streamreadablefromwebreadablestream-options here? Is the `@edge-runtime/node-utils` `toToReadable` doing something else?

Reply:
It's the same but supporting Node.js v14
https://github.com/vercel/edge-runtime/blob/main/packages/node-utils/src/edge-to-node/stream.ts#L11